Src/DeUrgenta.Certifications.Api/Storage/S3Storage.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Amazon.S3;
+using Amazon.S3.Model;
+using DeUrgenta.Certifications.Api.Storage.Config;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Extensions.Options;
+
+namespace DeUrgenta.Certifications.Api.Storage
+{
+    public class S3Storage : IBlobStorage
```

Comment:
I know there is some emulator for this in order to run this setup locally ,maybe we can add some documentation on that

Reply:
created #96 to do this